portfolio/src/main/java/com/google/sps/servlets/InitServlet.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.annotation.WebListener;
+import java.util.logging.Logger;
+import com.googlecode.objectify.ObjectifyService;
+import com.google.sps.data.Comment;
+
+/** Servlet that performs initial operations. */
+@WebListener
```

Comment:
Good call with the WebListener!

You could be a bit more specific in the documentation comment above about what those "initial operations" are.

Reply:
I thought it was better to be generic because for the moment this class is only used to register the Comment class but in general it can be used for any kind of initialization.

Reply:
I agree on not adding implementation details to a comment. But this would be more about logic of what has to be done on server startup like for example:
 - initialize database connection (this is not an implementation detail, the detail would be which database and what password is used or something like that)
 - (later maybe) initialize in-memory cache for comments